assets/tiled-project/tilemaps/animated_trees.tsx

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
According to format, this is tileset, not a tilemap, so it should not be in this folder. Also, please, use camelCaseFileNames.